content/blog/newsletter/2024-04-05-newsletter.md

```diff
@@ -0,0 +1,236 @@
+the options that are already available. If you are already self-hosting Snikket, or planning to, nothing is changing for you.
+Though please do [donate to support](https://snikket.org/donate/) the project, even a little helps!
+
+European Union news:
+  - [Designated gatekeepers must now comply with all obligations under the Digital Markets Act](https://ec.europa.eu/commission/presscorner/detail/en/ip_24_1342)
+  - [Interoperable Europe act: Council adopts new law for more efficient digital public services across the EU](https://www.consilium.europa.eu/en/press/press-releases/2024/03/04/interoperable-europe-act-council-adopts-new-law-for-more-efficient-digital-public-services-across-the-eu/)
+
+## Software News
+
+### Clients and Applications
+
+- The [JMP newsletter introduces the JMP eSIM Adapter](https://blog.jmp.chat/b/march-newsletter-2024), allowing users to use eSIMs from any provider in any device. It explains how to download and write eSIMs to the adapter, including using an open source Android app or PC/SC reader. Cheogram Android also receives an update with new features and a visual refresh. There's a clarification on contact integration, with Google now requiring the app claim to upload the contacts even though it does not.
+- [Converse 10.1.7](https://github.com/conversejs/converse.js/releases/tag/v10.1.7) with StropheJS fix for [XEP-0474: SASL SCRAM Downgrade Protection](https://xmpp.org/extensions/xep-0474.html).
+- [Psi+ 1.5.1716](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/), [Psi+ 1.5.1682](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/),
+[Psi+ 1.5.1669](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/) have been released.
+
+### Servers
+
+- Openfire has been released in [version 4.8.1](https://discourse.igniterealtime.org/t/openfire-4-8-1-release/93681).
+  - Openfire inVerse plugin [version 10.1.7.1 has been released](https://discourse.igniterealtime.org/t/openfire-inverse-plugin-version-10-1-7-1-released/93740).
+  - Openfire [PionTurn plugin](https://discourse.igniterealtime.org/t/pionturn-plugin-1-0-0-released/93757) 1.0.0 released! It creates a TURN/STUN server for Openfire.
+
+### Libraries & Tools
+
+- A new release of [go-xmpp 0.1.4](https://github.com/xmppo/go-xmpp/releases/tag/v0.1.4).
+- A new release of [go-sendxmpp 0.9.0](https://salsa.debian.org/mdosch/go-sendxmpp/-/releases/v0.9.0).
+- [overpush](https://github.com/mrusme/overpush): A self-hosted, drop-in replacement for Pushover, that uses XMPP as delivery method and offers the same API for submitting messages, so that existing setups (e.g. Grafana) can continue working and only require changing the API URL. [Release article](https://xn--gckvb8fzb.com/goodbye-pushover-hello-overpush/).
```

Comment:
```suggestion
- [overpush](https://github.com/mrusme/overpush): A self-hosted, drop-in replacement for Pushover, that uses XMPP as delivery method. It offers the same API for submitting messages, so that existing setups (e.g. Grafana) can continue working and only require changing the API URL. [Release article](https://xn--gckvb8fzb.com/goodbye-pushover-hello-overpush/).
```